### PR TITLE
Correct jmax

### LINKED ISSUE #61: Correct formulation of Jmax
By going over the 2007-paper, and coding up some of the equations in Jupyter (https://git.list.lu/remko.nijzink/lai_fpar/-/blob/master/notebooks/lai_equations.ipynb), I noticed that the calculation of Jmax in the VOM is dimensionally inconsistent. In the VOM, it is this code:

```
!     * (Out[310], derived from (3.26)) Temperature dependence of Jmax
      jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)) * (-273.d0  &
     &           + topt_ + 273.d0 * p_R_ * topt_)) / ((25.d0 + 273.d0  &
     &           * p_R_ * topt_) * (tair_h(th_) + 273.d0 * p_R_        &
     &           * topt_))) * ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
     &           + topt_)) / (25.d0 + 273.d0 * p_R_ * topt_))) * i_ha  &
     &           + i_hd) * jmax25g_d(:)) / ((-1.d0 + p_E ** ((i_hd     &
     &           * (273.d0 + tair_h(th_) - topt_)) / (tair_h(th_)      &
     &           + 273.d0 * p_R_ * topt_))) * i_ha + i_hd)
```

In Jupyter, this is:

jmax1

However, the old documentation, says this:

jmax3

Looks like the conversions from air temperature to kelvin have gone wrong with the parenthesis. I put the above also in Jupyter:

jmax2

The differences seem not too big fortunately:

jmax_graph

### PR DESCRIPTION
**Pull request by rcnijzink**
Closes https://github.com/schymans/VOM/issues/61

### CODE REVIEW

File: VOM_Fortran/VOM-code/transpmodel.f90 [hunk 1]
```diff
@@ -1246,14 +1246,12 @@ subroutine vom_hourly_init ()
...
-      jmaxt_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)) * (-273.d0  &
-     &           + topt_ + 273.d0 * p_R_ * topt_)) / ((25.d0 + 273.d0  &
-     &           * p_R_ * topt_) * (tair_h(th_) + 273.d0 * p_R_        &
-     &           * topt_))) * ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
-     &           + topt_)) / (25.d0 + 273.d0 * p_R_ * topt_))) * i_ha  &
-     &           + i_hd) * jmax25t_d(:)) / ((-1.d0 + p_E ** ((i_hd     &
-     &           * (273.d0 + tair_h(th_) - topt_)) / (tair_h(th_)      &
-     &           + 273.d0 * p_R_ * topt_))) * i_ha + i_hd)
+      jmaxt_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_))) / ((25.d0 + 273.d0)  &
+        &           * p_R_ * (tair_h(th_) + 273.d0) ))* ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
```
Comment (schymans) [inline VOM_Fortran/VOM-code/transpmodel.f90]: This line is too long. I would re-write as:
```
     jmaxt_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)))            & 
    &           / ((25.d0 + 273.d0) * p_R_ * (tair_h(th_) + 273.d0))) &
    &           * ((-1.d0 + p_E ** (-(i_hd * (-298.d0                 &
```

File: VOM_Fortran/VOM-code/transpmodel.f90 [hunk 2]
```diff
@@ -1271,14 +1269,12 @@ subroutine vom_hourly_init ()
...
-      jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)) * (-273.d0  &
-     &           + topt_ + 273.d0 * p_R_ * topt_)) / ((25.d0 + 273.d0  &
-     &           * p_R_ * topt_) * (tair_h(th_) + 273.d0 * p_R_        &
-     &           * topt_))) * ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
-     &           + topt_)) / (25.d0 + 273.d0 * p_R_ * topt_))) * i_ha  &
-     &           + i_hd) * jmax25g_d(:)) / ((-1.d0 + p_E ** ((i_hd     &
-     &           * (273.d0 + tair_h(th_) - topt_)) / (tair_h(th_)      &
-     &           + 273.d0 * p_R_ * topt_))) * i_ha + i_hd)
+      jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_))) / ((25.d0 + 273.d0)  &
+        &           * p_R_ * (tair_h(th_) + 273.d0) )) * ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
```
Comment (schymans) [inline VOM_Fortran/VOM-code/transpmodel.f90]: This line is too long. I would re-write as:
```
     jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)))            & 
    &           / ((25.d0 + 273.d0) * p_R_ * (tair_h(th_) + 273.d0))) &
    &           * ((-1.d0 + p_E ** (-(i_hd * (-298.d0                 &
```

**Review by schymans — CHANGES_REQUESTED** [review]
Thanks for fixing this! I would just adjust the line breaks a bit.

File: VOM_Fortran/VOM-code/transpmodel.f90 (hunk 2 above)
Comment (rcnijzink) [reply VOM_Fortran/VOM-code/transpmodel.f90]: Yes, agree, that looks better... Made changes accordingly.

**Review by schymans — APPROVED** [review]

**Pull request merged**